pysonos/events.py

```diff
@@ -186,11 +188,32 @@ def listen(self, ip_address):
             The port on which the event listener listens is configurable.
             See `config.EVENT_LISTENER_PORT`
         """
-        address = (ip_address, self.requested_port_number)
-        self._listener_thread = EventServerThread(address)
+        for port_number in range(self.requested_port_number,
+                                 self.requested_port_number + 100):
+            address = (ip_address, port_number)
+            try:
+                server = EventServer(address, EventNotifyHandler)
+                break
+            except OSError as oserror:
+                if oserror.errno == errno.EADDRINUSE:
+                    log.warning(
```

Comment:
Is this message even needed? The log below seems to contain the same information.

This is such a corner case that I guess it doesn't matter much either way :)

Reply:
Changed to debug.  Might be helpful later if someone has an issue